STEERING-COMMITTEE.md

```diff
@@ -7,8 +7,7 @@ aliases:
   - /contributing/steering-committee/
 ---
 
-The Knative Steering Committee (KSC) is the ultimate authority for the Knative
-project, and governs all aspects of the project.
+The Knative Steering Committee (KSC) governs all aspects of the project.
```

Comment:
I have the same question.

Reply:
yep -- can we please exactly enumerate what 'all' includes, including edge cases? It's fine if this list is long, but we should try to make it as complete as possible

Reply:
what are the semantics of removing 'ultimate authority' if the SC governs all aspects of the project? In which sense does this cut down the authority of the SC? I think it would be important to call that out, such that we know which cut-back we're approving :-)